docs/escape_support.md

```diff
@@ -0,0 +1,112 @@
+
+### CSI (Control Sequence Introducer) - `ESC [`
+
+| ESCAPE     | STATUS      | NOTE                                              |
+| ---------- | ----------- | ------------------------------------------------- |
+| ``CSI ` `` | IMPLEMENTED |                                                   |
+| `CSI @`    | IMPLEMENTED |                                                   |
+| `CSI A`    | IMPLEMENTED |                                                   |
+| `CSI a`    | IMPLEMENTED |                                                   |
+| `CSI B`    | IMPLEMENTED |                                                   |
+| `CSI b`    | IMPLEMENTED |                                                   |
+| `CSI C`    | IMPLEMENTED |                                                   |
+| `CSI c`    | PARTIAL     | No parameter support                              |
+| `CSI D`    | IMPLEMENTED |                                                   |
+| `CSI d`    | IMPLEMENTED |                                                   |
+| `CSI E`    | IMPLEMENTED |                                                   |
+| `CSI e`    | IMPLEMENTED |                                                   |
+| `CSI F`    | IMPLEMENTED |                                                   |
+| `CSI f`    | IMPLEMENTED |                                                   |
+| `CSI G`    | IMPLEMENTED |                                                   |
+| `CSI g`    | IMPLEMENTED |                                                   |
+| `CSI H`    | IMPLEMENTED |                                                   |
+| `CSI h`    | PARTIAL     | Only modes `4` and `20` are supported             |
+| `CSI ? h`  | PARTIAL     | Supported modes:                                  |
+|            |             |   `1`                                             |
+|            |             |   `3`                                             |
+|            |             |   `6`                                             |
+|            |             |   `7`                                             |
+|            |             |   `12`                                            |
+|            |             |   `25`                                            |
+|            |             |   `1000`                                          |
+|            |             |   `1002`                                          |
+|            |             |   `1004`                                          |
+|            |             |   `1005`                                          |
+|            |             |   `1006`                                          |
+|            |             |   `1007`                                          |
+|            |             |   `1049`                                          |
+|            |             |   `2004`                                          |
```

Comment:
I'm not entirely sure how this should be formatted, but it seems a bit weird that it looks like that in a table.

Reply:
Have you checked the output? I think it looks fine. I agree with you though that I'm not entirely sure if this is the best format, since it uses up a lot of vertical space while not utilizing horizontal space at all.

Do you have any suggestions?

Reply:
I was looking on rendered md file, yeah. maybe just collon separated list is fine instead of virtual layout.

Reply:
Colon separated? Why colon? I'd imagine if anything it should be comma separated?

Reply:
yeah, `comma` one.

Reply:
Does it look better to you now?